source/_components/egardia.markdown

```diff
@@ -100,22 +108,28 @@ There seem to be multiple versions of software running on GATE-02 devices; we ha
 2. Once logged in, go to *System Settings*, *Report* and change the Server Address for your primary server to the IP or hostname of your Home Assistant machine. You can leave the port number set to 52010 or change it to anything you like. **Make sure to change the settings of the primary server otherwise the messages will not come through. Note that this will limit (or fully stop) the number of alarm messages you will get through Egardia's / Woonveilig services.** Maybe, that is just what you want. Make sure to save your settings by selecting 'OK'.
 3. The Egardia component relies on capturing the status codes that your alarm emits when something happens (status change or trigger). These codes will be unique for every situation - i.e. the code emitted by the alarm when a sensor is triggered is unique to that sensor. Also, if you have multiple users or remotes, each remote has unique codes that are emitted by the alarm when status is changed using that remote or by that user. For the Egardia component to work correctly you will need to capture the codes. To do this, on your Home Assistant machine run `$ sudo python3 egardiaserver.py`. Refer to the [python-egardia repository](https://github.com/jeroenterheerdt/python-egardia) for detailed documentation on parameters. This will receive status codes from your alarm control panel and display them. Record the codes shown as well as the status they relate to (see step 4 below). Make sure to change the status of your alarm to all states (disarm, arm, home) by all means possible (all users, remotes, web login, app) as well as trigger the alarm in all ways possible to get 100% coverage of all the codes the alarm system generates. You will need to run this script once and stop it once you have captured all the possible codes. Also, if you ever add users, remotes or sensors to your alarm system, make sure to re-run the script to capture the extra codes so you can update your configuration (see step 4 below). **For comfort, before triggering the alarm it might be good to disable the siren temporarily (can be done in Panel Settings).**
 4. Once you have the codes, update your `configuration.yaml`:
-    ```yaml
-    # Example configuration.yaml entry
-    egardia:
-      host: YOUR_HOST
-      username: YOUR_USERNAME
-      password: YOUR_PASSWORD
-      report_server_enabled: true
-      report_server_port: PORT_OF_EGARDIASERVER (optional, defaults to 52010)
-      report_server_codes:
-        arm: XXXXXXXXXXXXXXXX, XXXXXXXXXXXXXXXX
-        disarm: XXXXXXXXXXXXXXXX, XXXXXXXXXXXXXXXX
-        armhome: XXXXXXXXXXXXXXXX
-        triggered: XXXXXXXXXXXXXXXX, XXXXXXXXXXXXXXXX, XXXXXXXXXXXXXXXX
-        ignore: XXXXXXXXXXXXXXXX
-    ```
+
```

Comment:
This should not have been changed. It breaks the list...

Reply:
I thought to do something well in the context of a markdownlint, I adjusted it.

Reply:
👯